plat/common/tbbr/plat_tbbr.c

```diff
@@ -7,7 +7,7 @@
 #include <assert.h>
 #include <auth/auth_mod.h>
 #include <platform.h>
-#include <platform_oid.h>
+#include <tbbr_oid.h>
```

Comment:
Please move the include to be in alphabetical order?
Also, please update the copyright header to 2016-2017.